### ISSUE TITLE
RPM Upgrade remove init scripts

### ISSUE THREAD

**Issue by Pinpin31** [TykTechnologies/tyk#1319]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
For upgrading rpm tyk-dashboard from 1.3.9 to 1.4.0, we use rpm upgrade with this command :
`rpm -U tyk-dashboard-1.4.0-1.x86_64.rpm`

The upgrade remove init scripts in /etc/init.d/

**What is the expected behavior?**

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
To reproduce the problem, start a docker container with Centos and install version 1.3.9 of Dashboard and then update to 1.4.0 :

```bash
docker run -v /home/login/Download:/tmp -it centos bash
[root@536261f5a9f8 tmp]# rpm -i tyk-dashboard-1.3.9-1.x86_64.rpm 
warning: tyk-dashboard-1.3.9-1.x86_64.rpm: Header V4 RSA/SHA1 Signature, key ID 729ea673: NOKEY
Installing init scripts...
Found Systemd
Found SysV1
Found Sysv2
[root@536261f5a9f8 tmp]# ls /etc/init.d/
README         tyk-dashboard  
[root@536261f5a9f8 tmp]# rpm -U tyk-dashboard-1.4.0-1.x86_64.rpm 
warning: tyk-dashboard-1.4.0-1.x86_64.rpm: Header V4 RSA/SHA1 Signature, key ID 729ea673: NOKEY
Installing init scripts...
Found Systemd
Found SysV1
Found Sysv2
Removing init scripts...
Found Systemd
Found SysV1
[root@536261f5a9f8 tmp]# ls /etc/init.d/
README
[root@536261f5a9f8 tmp]# ls /opt/tyk-dashboard/
CHANGELOG.md  README.md  lang    schemas        tyk_analytics.conf
EULA.md       install    portal  tyk-analytics  utils
[root@536261f5a9f8 tmp]#
```

Same problem with tyk-gateway RPM.

**Which versions of Tyk affected by this issue? Did this work in previous versions of Tyk?**
Dashbaord 1.4.0
Gateway 2.4.0

**Comment by excieve** [comment]
Thanks, the fix is coming up with packaging updates.

**Comment by excieve** [comment]
This will be fixed when #1330 lands and gets released. For now, forcing reinstall should bring the init scripts back.
